misc/docerina/src/main/java/org/ballerinalang/docgen/docs/utils/BallerinaDocUtils.java

```diff
@@ -58,6 +60,24 @@ public static void packageToZipFile(String sourceDirPath, String zipFilePath) th
         }
     }
 
+    public static byte[] checkHash(byte[] contentInBytes, String algorithm) {
```

Comment:
rename the function name to getHash

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/42386/commits/fff5836b373e5e37d3680640581edf928a6344c0